nemo_rl/models/policy/dtensor_policy_worker_v2.py

```
@@ -995,6 +1000,9 @@ def get_logprobs(
                         if len(vlm_kwargs) > 0:
                             del model_args["flash_attn_kwargs"]
 
+                        if 'flash_attn_kwargs' in model_args:
+                            del model_args['flash_attn_kwargs']
```

Ruff (0.12.2)

Use `pop` instead of `key in dict` followed by `del dict[key]`

Replace `if` statement with `.pop(..., None)`

(RUF051)